### PR TITLE
Add Redmi Note4 Mtk, Nubia z11 miniS and ZTE BA510 Support

### PR DESCRIPTION
**Pull request by ililaoban**

### CODE REVIEW

File: data/devices/xiaomi.yml [hunk 1]
```diff
@@ -1,4 +1,30 @@
 ---
+- name: Redmi Note4 (MTK)
```
Comment (chenxiaolong) [inline data/devices/xiaomi.yml]: Can you please change the name to include `Xiaomi`?

**Comment by ililaoban** [comment]
@chenxiaolong Ok ,I've done.

**Comment by chenxiaolong** [comment]
@chenxiaolong-bot test this please

**Comment by chenxiaolong** [comment]
@chenxiaolong-bot test this please

**Pull request merged**